tests/mcs/testutil.go

```diff
@@ -37,6 +37,13 @@ func SetupTSOClient(ctx context.Context, re *require.Assertions, endpoints []str
 	return cli
 }
 
+// SetupTSOClient creates a TSO client for test.
+func SetupClient(ctx context.Context, re *require.Assertions, endpoints []string, opts ...pd.ClientOption) pd.Client {
```

Comment:
Shall we change the previous one SetupTSOClient to SetupClientWithKeyspace?

Reply:
There is no specific TSO client now. Just the difference of w/ or w/o keyspace. I forgot to change SetupTSOClient to  SetupClientWithKeyspace.